### ISSUE TITLE
[BUG]: Installation of the COE in GCC High

### ISSUE THREAD

**Issue by cdt2021** [microsoft/powerapps-tools#839]
I'm attempting to install the "Center of Excellence - Core Components" version 1.98 in GCC High.  All of the connections are available except the "Notifications" Connector.  When I attempt to install the solution it tells me I need to update the notifications connector.  When I attempt to create the connection I get the following error:  "Could not find API 'shared_flowpush' ".  According to the documentation I found,  this connector is not available in GCC.
https://docs.microsoft.com/en-us/connectors/flowpush/ 

This is the only connector that is preventing me from installing the COE in GCC High.  Can someone provide some guidance on how I can install the COE in GCC High without the "Notification" Connector?  

![image](https://user-images.githubusercontent.com/80719563/111233096-587e1000-85ba-11eb-9367-3892c269bda6.png)

**Comment by manuelap-msft** [comment]
Hello,

thank you for letting us know, and sorry for the inconvenience this is causing! There are two options to proceed

1. A few other customers raised issues with the Notifications connector as well, so in our next release we will be removing the use of this connector. The next release is targeted for first week of April.
2. If Classic Experience is available in GCC High, you can import the solution through the Classic experience which won't prompt you to set connections for all connectors. Instead, you will set them after import by navigating to the Default Solution > Editing Connection References to set connections
![image](https://user-images.githubusercontent.com/55705669/111271827-a5e59600-8629-11eb-982e-662ae4caf453.png)

If you want to proceed with Option 2, let us know here and we can provide step by step instructions for this.

**Comment by cdt2021** [comment]
I working on a tight deadline so I would like to move forward with option #2.  I have "Switch to classic" as an available option. Please provide the instructions so that I can move forward.   How would you suggest to monitor the solutions and activities supported by the notifications service since it will not be enabled.  Also,  are there any other work arounds that I should know about in GCC High when implementing the COE?

**Comment by JeneferM-MSFT** [comment]
Are you upgrading? If so, from what version? Is there a feature in particular you need from this new release?

**Comment by cdt2021** [comment]
I'm not upgrading.  This is a new environment we're building from the ground up so the bare minimum is there at the moment.  I'm new to PowerApps and the GCC High Environment so I'm still learning about the features and functions.  I know there's some limitation of features in GCC High vs Commercial.  I wanted to make sure as I install the COE that I wasn't going to run into any additional issues like the issue with the notification service.   Please advise how to proceed.

**Comment by cdt2021** [comment]
Hello @manuelap-msft. Requesting Step by Step instructions for importing the COE Starterkit in GCC using the classic mode.

**Comment by JeneferM-MSFT** [comment]
I'm afraid I can't answer the generalized question about GCC/GCC High. We do have specialists in this area who give us feedback, and we believe we have all the spots ironed out for Core and GCC/GCC High.  And please note also that the toolkit is not officially supported. It is a starter kit to get folks going and we fix issues as budget allows. Though we do our best to help out as much as possible. 

All that said, this issue you found is just a regression, we rushed to satisfy one customer and it bit others. So we are reverting. 

I went ahead and made a local copy of core for you here with the issue addressed. You will do the rest of the steps in setup [(see setup instructions)](https://docs.microsoft.com/power-platform/guidance/coe/setup-core-components) as normal and just use this solution file instead of the one in GitHub.

You will want to be sure to update with the April release to get back into a more official state with a shipped version.

[CenterofExcellenceCoreComponents_1_99_managed.zip](https://github.com/microsoft/powerapps-tools/files/6157056/CenterofExcellenceCoreComponents_1_99_managed.zip)

**Comment by JeneferM-MSFT** [comment]
Now that we have you unblocked I will dupe to the bug tracking release of the fix to everyone else. 
Thanks for using CoE
Duplicate of #815
